src/status_im/contexts/wallet/events.cljs

```diff
@@ -240,10 +242,32 @@
                                                  accounts))
                                              stored-accounts
                                              tokens-per-account))]
-     {:db (-> db
+     {:fx [[:dispatch [:wallet/get-last-wallet-token-update-if-needed]]]
+      :db (-> db
               (update-in [:wallet :accounts] add-tokens tokens)
               (assoc-in [:wallet :ui :tokens-loading address] false))})))
 
+(rf/reg-event-fx
+ :wallet/get-last-wallet-token-update-if-needed
+ (fn [{:keys [db]}]
+   (when (->> (get-in db [:wallet :ui :tokens-loading])
+              vals
+              (every? false?))
+     {:fx [[:json-rpc/call
+            [{:method     "wallet_getLastWalletTokenUpdate"
```

Comment:
@alwx - I believe you need to update this PR branch to point to the `ab/issue-5987-last-token-update-time` status-go branch.